File: planner/app/src/main/java/com/pro3/planner/views/AddElementView.java
package com.pro3.planner.views;

import android.content.Context;
import android.widget.ArrayAdapter;

/**
 * Created by linus_000 on 14.11.2016.
 */

public class AddElementView extends ElementDialogView {

    public AddElementView(final Context context, final ArrayAdapter<CharSequence> categoryAdapter) {
        super(context, categoryAdapter);

        colorAdapter.setCheckedPosition(0);
        ((ColorElementView) colorAdapter.getView(0, null, null)).setChecked(true);
        selectedColor = colorAdapter.getItem(0).getColor();

    }
}

File: planner/app/src/main/java/com/sunilson/firenote/presentation/addElementDialog/AddElementView.kt
package com.sunilson.firenote.presentation.addElementDialog

import android.content.Context
import com.sunilson.firenote.presentation.shared.views.ColorElementView
import com.sunilson.firenote.presentation.shared.views.ElementDialogView

class AddElementView(context: Context) : ElementDialogView(context) {
    init {
        colorAdapter.setCheckedPosition(0)
        (colorAdapter.getView(0, null, null) as ColorElementView).isChecked = true
        selectedColor = colorAdapter.getItem(0)!!.color
    }
}